codespell_lib/data/dictionary.txt

```diff
@@ -23296,13 +23394,21 @@ execising->exercising
 execpt->except
 execption->exception
 execptions->exceptions
+exect->exact, expect, eject,
```

Comment:
Perhaps `elect`, `execute`, `exert` too?

Same with final `s` below.

Reply:
I'll add `exert`. I couldn't find the others.